### PR TITLE
method to create stablecoin

### PR DESCRIPTION
**Pull request by luissanchez0305**
method to set stable coin

### CODE REVIEW

**Review by molekilla — APPROVED** [review]
lg, fix conflicts

File: .gitignore [hunk 1]
```diff
@@ -101,4 +101,6 @@ dist
 .dynamodb/
 
 # TernJS port file
-.tern-port
\ No newline at end of file
+.tern-port
+
+build
```
Comment (zubietaroberto) [inline .gitignore:106]: The contracts you deploy should be commited to the repo.

**Review by zubietaroberto — CHANGES_REQUESTED** [review]

**Comment by luissanchez0305** [comment]
pushed builds

File: .gitignore (hunk 1 above)
Comment (luissanchez0305) [reply .gitignore]: listo

**Review by zubietaroberto — APPROVED** [review]

**Pull request merged**